packages/amplify-graphql-auth-transformer/src/utils/schema.ts

```diff
@@ -230,6 +231,17 @@ export const addDirectivesToField = (
   }
 };
 
+export const addDirectivesToObject = (
+  ctx: TransformerTransformSchemaStepContextProvider,
+  typeName: string,
+  directives: Array<DirectiveNode>,
+): void => {
+  const type = ctx.output.getType(typeName) as ObjectTypeDefinitionNode;
+  if (type) {
+    ctx.output.putType(extendObjectWithDirectives(type, directives));
+  }
+};
```

Comment:
Nit: This would break convention in this file...
```suggestion
export const addDirectivesToObject = (
  ctx: TransformerTransformSchemaStepContextProvider,
  typeName: string,
  directives: Array<DirectiveNode>,
): void => {
  const obj = ctx.output.getObject(typeName);
  if (obj) {
    ctx.output.updateObject(extendObjectWithDirectives(obj, directives));
  }
};
```